src/helpers/createFeatureAPI.js

```diff
@@ -1,7 +1,8 @@
 import core from 'core';
 import hotkeys from 'src/apis/hotkeys';
 import localStorageManager from 'helpers/localStorageManager';
-import TouchEventManager from 'helpers/TouchEventManager';
+import touchEventManager from 'helpers/TouchEventManager';
```

Comment:
seeing `hotkeysManager` is camel cased, should we do the same with `TouchEventManager` to make it consistent?